### DISCUSSION TITLE
Inclusion of Negative entities while fine tuning

### DISCUSSION THREAD

**Discussion by aayush-5** [urchade/GLiNER#129] [Q&A]
Hi @urchade and @all_community_people

Wonderful paper and models !!! In the paper , it is mentioned that in order to improve accuracy author have used Negative entities. Now if I want to improve the accuracy as per my dataset then how can i add negative entities in the training dataset while doing fine tuning ?

Thanks & Regards
Aayush Rah

**Comment by hitish** [comment]
I am also looking forward for using negative to improve efficiency but could not find any example how to use it . Do we need to start and end tokens along with negative entities??

**Comment by nadolsw** [comment]
Also interested in this!

**Comment by St4r4x** [comment]
@body_gliner_129.txt
